### DISCUSSION TITLE
No module named 'textual'

### DISCUSSION THREAD

**Discussion by protidev** [Textualize/textual#4115] [Q&A] [answered]
Hi! When im trying to open my file that useb textual i got an error and it closes, if someone can help, i didn't found any working method to solve
![image](https://github.com/Textualize/textual/assets/90475974/3ce3e31b-48f9-46b7-af88-6c18c7fe5c50)

**Comment by davep** [comment, marked as answer]
This would suggest you don't have Textual installed in the environment you're trying to run your application in.
